Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
Strange!  And adding `\s*` fixed it?

On my machine, the syntax there is innocuous, and colors are reasonable:
![screen shot 2016-04-18 at 4 55 14 pm](https://cloud.githubusercontent.com/assets/27570/14622335/6fba7cae-0586-11e6-9e91-29fc6b3bf942.png)